pkg/controllers/management/auth/crtb_handler.go

```diff
@@ -132,10 +208,13 @@ func (c *crtbLifecycle) reconcileSubject(binding *v3.ClusterRoleTemplateBinding)
 				break
 			}
 		}
+		addCondition(binding, condition, SubjectExists, binding.UserPrincipalName, nil)
 		return binding, nil
 	}
 
-	return nil, errors.Errorf("Binding %v has no subject", binding.Name)
+	err := errors.Errorf("Binding %v has no subject", binding.Name)
```

Comment:
https://github.com/pkg/errors has been archived, and is no longer supported, should these be converted to `fmt.Errorf()` with wrapping?

Reply:
Sure.

Reply:
Done with commit 8af1d5e2d89cad646ae04a68059a0ff3f8b7cb25